lib/jellyfish/room_service.ex

```diff
@@ -137,6 +167,34 @@ defmodule Jellyfish.RoomService do
     {:noreply, state}
   end
 
+  defp receive_resources(results \\ [], nodes \\ []) do
```

Comment:
But multicall doesn't solve a problem that process could die in the meantime. If you prefer it to be multicall I can change. 
As for RoomService being a GenServer if I remember correctly it was done to avoid a situation where a room would be created two times by different users, but I am not sure if this is still valid.

Reply:
I meant that multicall returns a list of bad nodes out of the box. We don't have to manually wait x seconds as in the previous solution :wink: